asreview/webapp/src/Components/HistoryDialog.js

```diff
@@ -18,7 +18,7 @@ import { makeStyles } from '@material-ui/core/styles';
 
 import ArrowBackIcon from '@material-ui/icons/ArrowBack';
 import FavoriteIcon from '@material-ui/icons/Favorite';
-import CloseIcon from '@material-ui/icons/Close';
+import ThumbDownIcon from '@material-ui/icons/ThumbDown';
```

Comment:
Interesting idea! 

If we are changing it here, then we should also change it in the review screen (and Prior Knowledge screen?).

Reply:
I think it makes sense...

Reply:
I'll make a screenshot for preview!